dapp/webpack.config.js

```diff
@@ -1,124 +1,116 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 const sveltePreprocess = require('svelte-preprocess');
-const webpack = require("webpack");
+const webpack = require('webpack');
```

Comment:
I think there's no real change to this file, I'm not against doing lint, but it should probably be done only to files that were really changed -- hopefully one day we'll add CI.

Reply:
Same as above